Sources/_InternalTestSupport/SkippedTestSupport.swift

```diff
@@ -0,0 +1,70 @@
+
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2014 - 2025 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import class Foundation.FileManager
+import class Foundation.ProcessInfo
+import Testing
+
+extension Trait where Self == Testing.ConditionTrait {
```

Comment:
Given `Testing.ConditionTrait ` conforms to `Trait`, can't we simply write this instead?
```swift
extension Testing.ConditionTrait {
```

Reply:
This file was a direct copy from `IntegartionTests/Sources/IntegrationTests`.  I want to avoid changing it drastically until #8223  is completed